app/views/patients/show.html.erb

```diff
@@ -1,135 +1,87 @@
+      <div class="w-full sm:w-3/4 md:w-3/5" style="min-height:12rem;">
+        <div class="sm:mr-2 md:mr-4 bg-gray-50 p-4 sm:p-6 md:p-8 h-full rounded mb-8 sm:mb-4 border border-gray-200">
+          <p class="text-2xl capitalize mb-2">
+            <%= @patient.full_name %>
+          </p>
+          <hr />
+          <p class="my-2 text-gray-600 py-1 mb-4">
+            <%= "#{@patient.facility.name},  #{@patient.facility.district}, #{@patient.facility.state}" %>
+          </p>
+          <div class="flex justify-between mt-8">
+            <p class="font-medium text-gray-500">
+              Last Visit :
+              <span class="block font-normal text-xl text-gray-900">1st March, 2021</span>
+            </p>
+            <p class="font-medium text-gray-500">
+              Next Visit :
+              <span class="text-xl block text-gray-900">11th April, 2021</span>
+            </p>
+          </div>
+        </div>
+      </div>
+      <div class="w-full sm:w-1/4 md:w-2/5 justify-between">
+        <div>
+          <%= link_to 'Add Health Information', disease_history_path, class: 'w-full mx-1 mb-4 md:text-lg inline-flex justify-center py-4 px-4 border border-transparent shadow-sm rounded-md text-white bg-indigo-600 hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500'%>
+        </div>
+        <div class="text-indigo-800 md:text-lg w-full mx-1 mb-4 inline-flex justify-center py-4 px-4 border border-transparent shadow-sm rounded-md text-white bg-indigo-100 hover:bg-indigo-200 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-200">
+          <img src="<%= asset_path 'icons/edit.svg' %>" class="mx-5" />
+          <%= link_to "Edit Personal Details", { action: "edit" } %>
+        </div>
+        <div>
+          <%= link_to 'Edit Family Details', family_details_path, class: 'text-indigo-800 md:text-lg w-full mx-1 inline-flex justify-center py-4 px-4 border border-transparent shadow-sm rounded-md text-white bg-indigo-100 hover:bg-indigo-200 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-200'%>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="row mt-8 ">
+    <div class="col">
+      <h3 class="text-3xl leading-6 font-medium text-gray-800 mb-8 pb-8 border-b">
+        Additional Information
+      </h3>
+
+      <div class="flex justify-between mb-8 bg-indigo-100 px-8 py-4 rounded-md">
+        <%= link_to "Personal Information", show_personal_details_path, class: "tab-label px-4 py-6 text-lg text-white bg-indigo-600 rounded hover:bg-indigo-800 w-3/12 text-center" %>
+        <%= link_to "Family Details", show_family_details_path, class: "tab-label px-4 py-6 text-lg text-white bg-indigo-600 rounded hover:bg-indigo-800 w-3/12 text-center" %>
+        <%= link_to "Disease History", show_disease_history_path, class: "tab-label px-4 py-6 text-lg text-white bg-indigo-600 rounded hover:bg-indigo-800 w-3/12 text-center" %>
+      </div>
```

Comment:
The styles of these links are not suitable for small screens(mobile).

Reply:
Made Responsive Deisgn